lighthouse-core/audits/service-worker.js

```diff
@@ -148,18 +148,21 @@ class ServiceWorker extends Audit {
       };
     }
 
-    // Include the detailed pass/fail checklist as a diagnostic.
+    // Include the SW details as diagnostic data.
+    const {scriptUrl, scopeUrl} = serviceWorkerUrls;
     /** @type {LH.Audit.Details.DebugData} */
     const details = {
       type: 'debugdata',
-      items: [serviceWorkerUrls],
+      scriptUrl,
+      scopeUrl,
     };
 
     const startUrlFailure = ServiceWorker.checkStartUrl(artifacts.WebAppManifest,
       serviceWorkerUrls.scopeUrl);
     if (startUrlFailure) {
       return {
         score: 0,
+        details,
```

Comment:
I think it makes sense to return details in this (corner) case as well.

While the audit fails because the SW doesn't control the `start_url` (or there isn't a manifest to give a `start_url`), there _is_ a SW controlling the tested page, so something like the original request in #9683 (which may care only about pages with SWs, not the exact parameters of LH's audit) could still find it useful to have that information.

Reply:
good point, this makes total sense! 👍